### ISSUE TITLE
Open webgl project in webview

### ISSUE THREAD

**Issue by kadirmenz** [gree/unity-webview#767]
Hi,
Im open my unity webgl project inside my main unity project with this webview package. It successfully works but when I try to open url in webgl project which I open from webview ask me choose browser to open but I wanna open it from webview. What should I do?

**Comment by KojiNakamaru** [comment]
> I try to open url in webgl project which I open from webview

How do you currently perform this? If you use `Application.OpenURL()`, it will open an external browser. In order to open a url in the current webview, you should use `webViewObject.LoadURL()`.

**Comment by kadirmenz** [comment]
I try both Application.OpenURL() and javascript features. I open this link to test: https://webview-dbcf1.web.app 
When I open this link on webview I cant exit with any button each button has different exit way. I also try to open about:blank to detect it from webview but I cant because when I open it from webview it says choose browser to open it. 
I would be glad if you have any advice.

**Comment by KojiNakamaru** [comment]
Trying to reproduce the issue, I built a simple unity WebGL app with the following jslib/c# definitions

```javascript
mergeInto(LibraryManager.library, {
    MyLoadURL: function(href) {
        window.location.href = Pointer_stringify(href);
    }
});
```

```c#
    [DllImport("__Internal")]
    private static extern void MyLoadURL(string href);

    public void OnClick()
    {
        MyLoadURL("https://google.com");
    }
```

and displayed it in the sample app's webview (where the sample app is running on Android). When the button was pressed, the webview correctly transited to google.com.

**Comment by kadirmenz** [comment]
I actually try this one but let me try same like yours,
I wanna ask one more question sir, Is there any possibility to send message from unitywebgl to webview which is inside main unity app ?

**Comment by kadirmenz** [comment]
> 
Yes this works but when I try to redirect "about:blank" it ask me choose browser.
Actually I wanna close webview via this webgl build window.close() doesnt work so I think one way which is redirect to about:blank and is onStart see coming url is about:blank should close webview but if you have any good idea for it I would love to hear

**Comment by KojiNakamaru** [comment]
cf. https://github.com/gree/unity-webview/issues/769#issuecomment-1011307642
